src/AccessibilityTranslator.js

```diff
@@ -1,11 +1,18 @@
-import  _ from 'lodash';
+import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ROLES, getTagNameForDisplayObject } from './Roles.js';
-import AccessibilityObject from './RoleObjects/AccessibilityObject.js';
+import { ROLES, getTagNameForDisplayObject } from './Roles';
 
 /**
- * Update process for translating accessibility information for a stage to a DOM approach that is available to assistive technologies.  Applications should create an instance of this for each stage and provide instances with different parent nodes.  Since the drawing order may not always be convient for accessibility, this manages a separate tree of DisplayObjects.  After the instance is created, the setter for 'root' should be used to specify which DisplayObject will serve as the root of the accessibility tree and other DisplayObjects can be added to that to be included in the accessibility output.  This also helps minimize the processing done by this class along with reduce its output to the DOM.
+ * Update process for translating accessibility information for a stage to a DOM approach
+  that is available to assistive technologies.  Applications should create an instance
+  of this for each stage and provide instances with different parent nodes.  Since the
+  drawing order may not always be convient for accessibility, this manages a separate
+  tree of DisplayObjects.  After the instance is created, the setter for 'root' should
+  be used to specify which DisplayObject will serve as the root of the accessibility tree
+  and other DisplayObjects can be added to that to be included in the accessibility output.
+  This also helps minimize the processing done by this class along with reduce
+  its output to the DOM.
```

Comment:
Nit: multiline function comments should all start with a `*` that is aligned with the first `*` of `/*` that started the comment

Reply:
updated